src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,167 @@
+package duke;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.File;
+
+import duke.task.Task;
+import javafx.application.Application;
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+import javafx.scene.layout.Region;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+
+
+import java.util.Scanner;
+
+import duke.Parser;
+import duke.TaskList;
+
+/**
+ * duke.Main class of the program.
+ *
+ * @author Tania Tan Shu Qi
+ */
+public class Duke extends Application {
+
+    private ScrollPane scrollPane;
+    private VBox dialogContainer;
+    private TextField userInput;
+    private Button sendButton;
+    private Scene scene;
+    private Image userPic = new Image(this.getClass().getResourceAsStream("/images/DaCHOCO.jpg"));
+    private Image eueu = new Image(this.getClass().getResourceAsStream("/images/DaEUEU.jpg"));
+    public Duke() {
+
+    }
```

Comment:
If no need for one, I think it's better to leave out the empty constructor

Reply:
Oops, I just got to the JavaFX. Turns out there's a good reason for this, my mistake!